prac_09/unreliable_car_test.py

```diff
@@ -0,0 +1,4 @@
+from unreliable_car import UnreliableCar
+
+old_ute = UnreliableCar('Old Ute', 100, 40)
+print(old_ute.drive(20))
```

Comment:
With randomness, only 1 test is very hard to evaluate. Does it always do it like in this run? You need much more rigorous testing than this.